CHANGELOG.md

```diff
@@ -11,6 +11,19 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 VAST now supports the XDG base directory specification: The `vast.conf` is
+  now found at `${XDG_CONFIG_HOME:-${HOME}/.config}/vast/vast.conf`, and schema
+  files at `${XDG_DATA_HOME:-${HOME}/.local/share}/vast/schema/`. The
+  user-specific configuration file takes precedence over the global
+  configuration file in `${sysconfdir}/vast/vast.conf`.
+  [#1036](https://github.com/tenzir/vast/pull/1036)
```

Comment:
Is `${sysconfdir}` an environment variable? At least this syntax, next to `${HOME}`, suggests that.

Reply:
It's not. You can set a combination of `CMAKE_INSTALL_PREFIX` (default: `/usr/local`) and `CMAKE_INSTALL_SYSCONFDIR` (default: `share`) to change it during compile time.

Would you prefer `<sysconfdir>` over `${sysconfdir}`?

Reply:
I've gone ahead with my proposed change.